.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
project files of the IDE (inteliIDEA) should not be part of the repository, because they contains specific settings for your warking station, they can be add .gitignore file. Later when you get to the build tools chapter, you will find some new tools that are used